### PR TITLE
Fix typos in kubefed

### PR DESCRIPTION
**Pull request by m1093782566**
**What this PR does / why we need it**:

Fix some typos in kubefed.

**Which issue this PR fixes** : fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: federation/pkg/kubefed/init/init_test.go [hunk 1]
```diff
@@ -496,7 +496,7 @@ func TestCertsTLS(t *testing.T) {
 
 	for i, tc := range testCases {
 		// Make a copy of the client config before modifying it.
-		// We can't do a regular pointer deref shallow copy because
```
Comment (madhusudancs) [inline federation/pkg/kubefed/init/init_test.go]: This is intentional and a well understood abbreviation. I would leave it as it was before.

File: federation/pkg/kubefed/init/init_test.go [hunk 2]
```diff
@@ -579,7 +579,7 @@ func TestCertsHTTPS(t *testing.T) {
...
```
Comment (madhusudancs) [inline federation/pkg/kubefed/init/init_test.go]: Same as above.

File: federation/pkg/kubefed/join.go [hunk 3]
```diff
@@ -244,7 +244,7 @@ func (j *joinFederation) Run(f cmdutil.Factory, cmdOut io.Writer, config util.Ad
 			// 1. We want to create the secret in a specific namespace, which
 			//    is neither the "default" namespace nor the one specified
 			//    via the `--namespace` flag.
-			// 2. `SecretGeneratorV1` requires LiteralSources in a string-ified
```
Comment (madhusudancs) [inline federation/pkg/kubefed/join.go]: This is intentional too.  It means making something a string. I would leave this as it was before too.

**Review by madhusudancs — CHANGES_REQUESTED** [review]

File: federation/pkg/kubefed/init/init_test.go (hunk 1 above)
Comment (m1093782566) [inline federation/pkg/kubefed/init/init_test.go]: oops, I didn't realize it. It's fixed now. PTAL. Thnaks!

File: federation/pkg/kubefed/init/init_test.go (hunk 2 above)
Comment (m1093782566) [inline federation/pkg/kubefed/init/init_test.go]: oops, I didn't realize it. It's fixed now. PTAL. Thnaks!

File: federation/pkg/kubefed/join.go (hunk 3 above)
Comment (m1093782566) [inline federation/pkg/kubefed/join.go]: oops, I didn't realize it. It's fixed now. PTAL. Thnaks!

**Comment by m1093782566** [comment]
Glad to receive your feedback, @madhusudancs 

Comments are fixed now. Could you please take a look again?

**Comment by m1093782566** [comment]
ping @madhusudancs ~

**Review by madhusudancs — APPROVED** [review]

**Pull request merged**